File: backend/backend/src/main/java/com/backend/backend/Communication/Response/LoginResponse.java
package com.backend.backend.Communication.Response;

import java.util.UUID;

public class LoginResponse {

    private UUID id;

    private String username;

    private String email;

    private String jwtToken;

    public UUID getId() {
        return id;
    }

    public void setId(UUID id) {
        this.id = id;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getJwtToken() {
        return jwtToken;
    }

    public void setJwtToken(String jwtToken) {
        this.jwtToken = jwtToken;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public LoginResponse(UUID id, String username, String email, String jwtToken) {
        this.id = id;
        this.username = username;
        this.email = email;
        this.jwtToken = jwtToken;
    }
}

File: ClientApp/app/SzambiztHfApplibrary/src/main/java/com/e/szambizthfapplibrary/network/Response/LoginResponse.kt
package com.e.szambizthfapplibrary.network.Response

import java.util.*

class LoginResponse {
    private lateinit var id: UUID

    private lateinit var username: String

    private lateinit var email: String

    private lateinit var jwtToken: String

    fun getId(): UUID {
        return id
    }

    fun setId(id: UUID) {
        this.id = id
    }

    fun getUsername(): String {
        return username
    }

    fun setUsername(username: String) {
        this.username = username
    }

    fun getJwtToken(): String {
        return jwtToken
    }

    fun setJwtToken(jwtToken: String) {
        this.jwtToken = jwtToken
    }

    fun getEmail(): String {
        return email
    }

    fun setEmail(email: String) {
        this.email = email
    }

    fun LoginResponse(id: UUID, username: String, email: String, jwtToken: String) {
        this.id = id
        this.username = username
        this.email = email
        this.jwtToken = jwtToken
    }
}